v2/cmd/integration-test/http.go

```diff
@@ -33,6 +33,7 @@ var httpTestcases = map[string]testutils.TestCase{
 	"http/raw-dynamic-extractor.yaml":               &httpRawDynamicExtractor{},
 	"http/raw-get-query.yaml":                       &httpRawGetQuery{},
 	"http/raw-get.yaml":                             &httpRawGet{},
+	"http/raw-path.yaml":                            &httpRawPath{},
```

Comment:
same as before, the filename should mention the forward slash, along with the test name